src/integrations/slack.py

```diff
@@ -1,28 +1,55 @@
-from ..clients.rest import RestClient
-from ..config import EnvManager
+from typing import List
+
+from src.clients import RestClient
+from src.config import EnvManager
 
 
 class SlackIntegration:
```

Comment:
Maybe we can rename this class to SlackBotIntegration or SlackApiIntegration

Reply:
Done!